*/*: remove unused patches

mm1ke opened the pull request:
another patch cleanup PR

---

Please check all the boxes that apply:

- [x] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [x] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [x] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [x] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

thesamesam commented:
For bash, 92328f4b82f4d1c6c48a6507ee5a81e13cbb660f wasn't right: reverted in 4a887696652aa1679fc688141dd8b5f167bb2356 and fixed in 3efd3dcbf81e03f6868338233526f9d306bccfef.

mm1ke commented:
> For bash, [92328f4](https://github.com/gentoo/gentoo/commit/92328f4b82f4d1c6c48a6507ee5a81e13cbb660f) wasn't right: reverted in [4a88769](https://github.com/gentoo/gentoo/commit/4a887696652aa1679fc688141dd8b5f167bb2356) and fixed in [3efd3dc](https://github.com/gentoo/gentoo/commit/3efd3dcbf81e03f6868338233526f9d306bccfef).

Thx @thesamesam and sorry for the inconvenience

thesamesam commented:
np
